parm/config/gfs/config.resources

```diff
@@ -73,6 +73,14 @@ elif [[ "${step}" = "preplandobs" ]]; then
     npe_node_preplandobs=1
     export npe_node_preplandobs
 
+elif [[ "${step}" = "prepiodaobs" ]]; then
+    export wtime_prepiodaobs="00:05:00"
+    npe_prepiodaobs=1
+    export npe_prepiodaobs
+    export nth_prepiodaobs=1
+    npe_node_prepiodaobs=1
+    export npe_node_prepiodaobs
```

Comment:
I know this script is inconsistent in general, including the `preplandobs` directly above it that uses this identical pattern, but can we at least make this new section use a consistent style? Two variables are defined then exported, another is exported as it is defined.